app/controllers/users_controller.rb

```diff
@@ -273,6 +273,7 @@ def user_params
       :message_on_new_phenotype,
       :message_on_phenotype_comment_reply,
       :message_on_snp_comment_reply,
+      user_phenotypes_attributes: [:id, :variation, :_destroy]
```

Comment:
OK I tried it now - when I delete the user-phenotype via the edit phenotype form I get this error in the logs:

`Unpermitted parameter: _destroy`

and it doesn't get deleted and reappears on refresh, so I think I'll keep it!